### ISSUE TITLE
[CI jobs] More tests need to run in CI

### ISSUE THREAD

**Issue by rdzhabarov** [pytorch/glow#1257]
Glow contains a variety of validation tests:
1) Unit tests, these tests are run as part of the CI currently and verify inference as well as training
2) Larger tests that exercise training/backpropagation (cifar10, mnist, ptb) and typically take a very long time to execute. The execution time can be tuned by introducing a dynamic number of iterations if needed.
3) Inference on well known pre-trained models: resnet50, vgg19, squeezenet, shufflenet, etc. Model definitions that are in ONNX or Caffe2 format are required for those tests. You can check on how those models downloaded for [ONNX](https://github.com/pytorch/glow/blob/master/utils/download_onnx_models.sh) and [Caffe2](https://github.com/pytorch/glow/blob/master/utils/download_caffe2_models.sh). These tests do not assert current results against expected and used in a manual validation. Sample input/output look like 
 ```
      Model: resnet50/predict_net.pb
      File: tests/images/imagenet/cat_285.png Result:285
```

The goal here is to introduce a script which allows running (3) on CI.
Few things to consider before implementing:
* Measure how much time it takes to download weights/models and run validation
* Depending on time, enable validation on every PR or as part of the daily travis cron job
* Create a script which makes test fail in case output mismatch with the expected output (run this on CI)

Important scenarios include AOT examples, which should be validated on the CI in a similar way as we propose to do for (3). AOT examples also require downloading weights and models.

**Comment by bertmaher** [comment]
Seems like the LIT stuff has solved this!
